rspotify-model/src/track.rs

```diff
@@ -104,14 +92,7 @@ pub struct SavedTrack {
 }
 
 /// Track id with specific positions track in a playlist
-pub struct TrackPositions<'id> {
-    pub id: &'id TrackId,
+pub struct TrackPositions {
+    pub id: TrackId,
     pub positions: Vec<u32>,
 }
-
```

Comment:
Is it ok to remove this `new` constructor function?

Reply:
Yes, I don't get why it was needed in the first place if all the fields are public. You can just construct it with `TrackPostitions { id, positions }` just like how it works for every other model.